Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs

```diff
@@ -114,7 +114,8 @@ public string ContextTitle
         [Category(@"Appearance")]
         [Description(@"Display color for associated contextual tabs.")]
         [DefaultValue(typeof(Color), "Red")]
-        public Color ContextColor
+        //[DisallowNull]
+        public Color? ContextColor
```

Comment:
Do not do this, Colour should never be null or nullable

Reply:
Can I remove `(value == null) || `?

Reply:
yes, and remove the nullable, and put the [DisallowNull] back in

Reply:
Done